src/lib/components/Alert/index.tsx

```diff
@@ -4,11 +4,11 @@ import { HiX } from 'react-icons/hi';
 
 export type AlertProps = PropsWithChildren<{
   color?: 'blue' | 'red' | 'green' | 'yellow' | 'gray';
-  Icon?: FC<ComponentProps<'svg'>>;
+  icon?: FC<ComponentProps<'svg'>>;
   rounded?: boolean;
   withBorderAccent?: boolean;
   additionalContent?: ReactNode;
-  onDismiss?: () => void;
+  onDismiss?: boolean | (() => void);
```

Comment:
I think the previous type was more suitable!

Reply:
As documented, I made the change to fix broken story behavior, not because I think the type is more appropriate.